apng-drawable/src/main/kotlin/com/linecorp/apng/decoder/Apng.kt

```diff
@@ -72,7 +73,9 @@ internal class Apng(
         }
 
     init {
+        Trace.beginSection("Apng#draw")
```

Comment:
This method invocation may not be optimized by proguard.
How about to enclose with `if (BuildConfig.DEBUG) { }`?

Reply:
Trace.beginSection method will collect timing information **[only while system is running systrace](https://android.googlesource.com/platform/frameworks/base/+/pie-release/core/java/android/os/Trace.java#288).** So it seems that no need to be optimized by proguard IMHO.
In case of Trace API usage on android platform code, there are no such debug block. (See : https://cs.bazel.build/search?q=r%3Adroid+lang%3Ajava+Trace%5C.traceBegin or https://cs.bazel.build/search?q=r%3Adroid+lang%3Ajava+Trace%5C.begin )

And, while BuildConfig.DEBUG is on (via debug build), cmake will turn off optimization (using -O0 ).   Therefore, Trace api cannot collect proper timing.

Reply:
Thank you for the detailed advices!